.idea/misc.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="false" project-jdk-name="11" project-jdk-type="JavaSDK" />
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="11" project-jdk-type="JavaSDK" />
 </project>
```

Comment:
for .idea/ these two file can you do the rebase to catch the newest master code. I think you won't have these file changed. [git rebase](https://medium.com/osedea/git-rebase-powerful-command-507bbac4a234)